<launch>

  <node pkg="floor_plane_bayesian" type="floor_plane_bayesian" name="floor_plane_bayesian" output="screen">
      launch-prefix="xterm -e gdb --args"
      <param name="base_frame" value="/bubbleRob" />
      <param name="max_range" value="2.0" />

      <remap from="/floor_plane_bayesian/scans" to="/depth_registered/points"/>
  </node>
</launch>
